striker-ui/Makefile.am

```diff
@@ -1,136 +1,66 @@
-EXTRA_DIST		= \
-			next.config.js \
-			next-env.d.ts \
-			tsconfig.json \
-			theme/index.ts \
-			styles/globals.css \
-			.lintstagedrc.json \
-			env.development \
-			package.json \
-			lib/extended_date/ExtendedDate.ts \
-			lib/fetchers/putJSON.ts \
-			lib/fetchers/fetchJSON.ts \
-			lib/fetchers/periodicFetch.ts \
-			lib/sanitizers/hostsSanitizer.ts \
-			lib/consts/ICONS.ts \
-			lib/consts/DEFAULT_THEME.ts \
-			lib/consts/SERVERS.ts \
-			lib/consts/ANVILS.ts \
-			lib/consts/API_BASE_URL.ts \
-			lib/consts/IS_DEV_ENV.ts \
-			lib/consts/NODES.ts \
-			.prettierrc.json \
-			types/AnvilServers.d.ts \
-			types/GetAllAnvilResponse.d.ts \
-			types/AnvilMemory.d.ts \
-			types/FetchResponse.d.ts \
-			types/AnvilSharedStorage.d.ts \
-			types/AnvilNetwork.d.ts \
-			types/AnvilTypes.d.ts \
-			types/GetResponse.d.ts \
-			types/AnvilNodeStatus.d.ts \
-			types/AnvilList.d.ts \
-			types/AnvilCPU.d.ts \
-			types/GetOneAnvilResponse.d.ts \
-			types/AnvilSet.d.ts \
-			types/AnvilStatus.d.ts \
-			types/AnvilReplicatedStorage.d.ts \
-			types/AnvilFileSystems.d.ts \
-			types/NodeSet.d.ts \
-			components/Memory.tsx \
-			components/Decorator.tsx \
-			components/Hosts/index.tsx \
-			components/Hosts/AnvilHost.tsx \
-			components/Spinner.tsx \
-			components/AnvilDrawer.tsx \
-			components/Text/HeaderText.tsx \
-			components/Text/index.tsx \
-			components/Text/BodyText.tsx \
-			components/FileSystem/index.tsx \
-			components/FileSystem/FileSystemsHost.tsx \
-			components/FileSystem/FileSystems.tsx \
-			components/Anvils/index.tsx \
-			components/Anvils/sortAnvils.ts \
-			components/Anvils/Anvil.tsx \
-			components/Anvils/AnvilList.tsx \
-			components/Anvils/SelectedAnvil.tsx \
-			components/Panels/index.tsx \
-			components/Panels/InnerPanel.tsx \
-			components/Panels/PanelHeader.tsx \
-			components/Panels/Panel.tsx \
-			components/Servers.tsx \
-			components/Storage.tsx \
-			components/AnvilContext.tsx \
-			components/SharedStorage/SharedStorage.tsx \
-			components/SharedStorage/index.tsx \
-			components/SharedStorage/SharedStorageHost.tsx \
-			components/Header.tsx \
-			components/Network/index.tsx \
-			components/Network/Network.tsx \
-			components/Network/processNetwork.ts \
-			components/CPU.tsx \
-			components/Bars/index.tsx \
-			components/Bars/ProgressBar.tsx \
-			components/Bars/AllocationBar.tsx \
-			pages/index.tsx \
-			pages/_app.tsx \
-			pages/_document.tsx \
-			.eslintrc.json \
-			commitlint.config.js \
-			.eslintignore \
-			package-lock.json
+outindexfile		= index.html
+outjsmodulesdir		= _next
+outimagesdir		= pngs
 
-htmldir			= $(localstatedir)/www/html
-outdir			= out
+# The build output directory listed under EXTRA_DIST contains the files to be
+# installed; there is no build process required.
+#
+EXTRA_DIST		= $(nextoutdir)
 
-# Trigger build target on make call without parameters.
-all: out
+# Blank target; the build output directory should be usable as-is.
+$(nextoutdir): ;
```

Comment:
I never saw this format before. What does it do?

Reply:
According to https://www.gnu.org/software/make/manual/html_node/Empty-Recipes.html, it's an empty target to avoid accidentally triggering instructions. In this context, it's for avoiding changes to the `out/` directory.